src/common/utils.ts

```diff
@@ -23,7 +24,7 @@ export const asArray = (args: string | string[]): string[] => {
 
 export const removeBlankAttributes = (obj: Record<string, any>): Record<string, any> => {
   const options: CleanOptions = {
-    emptyArrays: false,
+    emptyArrays: true,
```

Comment:
Please explain why?

Reply:
to stay consistent between core and api to avoid diff